build.gradle

```diff
@@ -0,0 +1,43 @@
+group 'compiler-fleet'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.2.21'
```

Comment:
You may want to replace the minor version with + to use the latest version: '1.2.+'